[2024-11-28T11:37:36.989+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: snowflake_demo_dag.insert_data manual__2024-11-28T11:37:31.781709+00:00 [queued]>
[2024-11-28T11:37:36.993+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: snowflake_demo_dag.insert_data manual__2024-11-28T11:37:31.781709+00:00 [queued]>
[2024-11-28T11:37:36.993+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2024-11-28T11:37:36.993+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 11
[2024-11-28T11:37:36.994+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2024-11-28T11:37:37.000+0000] {taskinstance.py:1383} INFO - Executing <Task(SnowflakeOperator): insert_data> on 2024-11-28 11:37:31.781709+00:00
[2024-11-28T11:37:37.003+0000] {standard_task_runner.py:55} INFO - Started process 403 to run task
[2024-11-28T11:37:37.005+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'snowflake_demo_dag', 'insert_data', 'manual__2024-11-28T11:37:31.781709+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/snowflake_dag.py', '--cfg-path', '/tmp/tmp_h1k_ocs']
[2024-11-28T11:37:37.005+0000] {standard_task_runner.py:83} INFO - Job 4: Subtask insert_data
[2024-11-28T11:37:37.034+0000] {task_command.py:376} INFO - Running <TaskInstance: snowflake_demo_dag.insert_data manual__2024-11-28T11:37:31.781709+00:00 [running]> on host 0270f4f4bd85
[2024-11-28T11:37:37.066+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=snowflake_demo_dag
AIRFLOW_CTX_TASK_ID=insert_data
AIRFLOW_CTX_EXECUTION_DATE=2024-11-28T11:37:31.781709+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-11-28T11:37:31.781709+00:00
[2024-11-28T11:37:37.067+0000] {sql.py:265} INFO - Executing: 
INSERT INTO demo_table (id, name, created_at)
VALUES (1, 'Airflow Demo', CURRENT_TIMESTAMP);
[2024-11-28T11:37:37.071+0000] {base.py:71} INFO - Using connection ID 'snowflake_conn_id' for task execution.
[2024-11-28T11:37:37.347+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/snowflake/connector/options.py:109: UserWarning: You have an incompatible version of 'pyarrow' installed (6.0.1), please install a version that adheres to: 'pyarrow<10.1.0,>=10.0.1; extra == "pandas"'
  "pyarrow", installed_pyarrow_version, pandas_pyarrow_extra

[2024-11-28T11:37:37.484+0000] {base.py:71} INFO - Using connection ID 'snowflake_conn_id' for task execution.
[2024-11-28T11:37:37.484+0000] {connection.py:300} INFO - Snowflake Connector for Python Version: 3.0.4, Python Version: 3.7.15, Platform: Linux-6.10.4-linuxkit-aarch64-with-debian-11.5
[2024-11-28T11:37:37.485+0000] {connection.py:1013} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-28T11:37:37.485+0000] {connection.py:1030} INFO - Setting use_openssl_only mode to False
[2024-11-28T11:37:37.904+0000] {cursor.py:806} INFO - query: [ALTER SESSION SET autocommit=False]
[2024-11-28T11:37:38.007+0000] {cursor.py:819} INFO - query execution done
[2024-11-28T11:37:38.007+0000] {cursor.py:962} INFO - Number of results in first chunk: 1
[2024-11-28T11:37:38.009+0000] {sql.py:375} INFO - Running statement: INSERT INTO demo_table (id, name, created_at)
VALUES (1, 'Airflow Demo', CURRENT_TIMESTAMP);, parameters: None
[2024-11-28T11:37:38.009+0000] {cursor.py:806} INFO - query: [INSERT INTO demo_table (id, name, created_at) VALUES (1, 'Airflow Demo', CURRENT...]
[2024-11-28T11:37:38.382+0000] {cursor.py:819} INFO - query execution done
[2024-11-28T11:37:38.383+0000] {sql.py:384} INFO - Rows affected: 1
[2024-11-28T11:37:38.384+0000] {snowflake.py:391} INFO - Rows affected: 1
[2024-11-28T11:37:38.384+0000] {snowflake.py:392} INFO - Snowflake query id: 01b8ac19-0001-3903-0006-ac5e0001a99e
[2024-11-28T11:37:38.384+0000] {cursor.py:806} INFO - query: [COMMIT]
[2024-11-28T11:37:38.616+0000] {cursor.py:819} INFO - query execution done
[2024-11-28T11:37:38.618+0000] {cursor.py:962} INFO - Number of results in first chunk: 1
[2024-11-28T11:37:38.619+0000] {connection.py:605} INFO - closed
[2024-11-28T11:37:38.671+0000] {connection.py:608} INFO - No async queries seem to be running, deleting session
[2024-11-28T11:37:38.772+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=snowflake_demo_dag, task_id=insert_data, execution_date=20241128T113731, start_date=20241128T113736, end_date=20241128T113738
[2024-11-28T11:37:38.807+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2024-11-28T11:37:38.823+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
